### ISSUE TITLE
Opacity bug going from 0.152.1 to 0.152.2

### ISSUE THREAD

**Issue by frankxw** [playcanvas/engine#15]
I'm not sure if this is the right place to report this, but I ran into an opacity issue earlier today.  I have a plane with an opacity map that was working yesterday and this morning, but stopped working at some point in the day.  I tracked it down to the PlayCanvas version getting bumped from 0.152.1 to 0.152.2  (exporting the pack and then using 0.152.1 instead makes to opacity work correctly).

Basically, it's not blending correctly except at certain angles.  I've got a scene set up with a "Clouds" plane above a city.  [See this album](http://imgur.com/a/93eUM)
